hw09-jdbc/src/main/java/ru/otus/hw09/jdbc/DbExecutor.java

```diff
@@ -0,0 +1,107 @@
+package ru.otus.hw09.jdbc;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+import java.sql.*;
+import java.util.List;
+import java.util.Optional;
+
+public class DbExecutor<T> {
+    private static Logger logger = LoggerFactory.getLogger(DbExecutor.class);
+
+    private Connection connection;
+    private final Class<T> clazz;
+    private final String selectStmt;
+    private final String insertStmt;
+    private final String updateStmt;
+
+    public DbExecutor(Connection connection, Class<T> clazz) {
+        this.connection = connection;
+        this.clazz = clazz;
+        this.selectStmt = ReflectionHelper.createSelectStatementForClass(this.clazz);
+        this.insertStmt = ReflectionHelper.createInsertStatementForClass(this.clazz);
+        this.updateStmt = ReflectionHelper.createUpdateStatementForClass(this.clazz);
+    }
+
+    public void create(T objectData) throws SQLException, IllegalAccessException, NoSuchFieldException {
```

Comment:
```suggestion
    public void create(T objectData) throws DbExecutorException {
```
Все эти исключения хорошо бы обернуть в свое, например, `DbExecutorException`.